scripts/instance.template.env

```diff
@@ -42,6 +42,16 @@ APIML_SECURITY_AUTH_PROVIDER=zosmf
 # List of discovery service URLs separated by comma
 ZWE_DISCOVERY_SERVICES_LIST=https://${ZOWE_EXPLORER_HOST}:${DISCOVERY_PORT}/eureka/
 
+####################
+# caching service
+# TCP port of caching service
+ZWE_CACHING_SERVICE_PORT=7555
+# specify persistent method of caching service
+# currently VSAM is the only option
+ZWE_CACHING_SERVICE_PERSISTENT=VSAM
+# specify the same data set name created by ZWECSVSM jcl
```

Comment:
`# specify the data set name of the caching service VSAM`
you don't specify names of other setup jobs as reference, so don't do it here either